CODEOWNERS

```diff
@@ -30,3 +30,7 @@ cloud-sql @kurtisvg
 healthcare @noerog
 iot @gguuss
 jobs @happyhuman
+
+#Composer samples
+@sofisl
```

Comment:
Sorry - I was unclear how this should be formatted before! I think we don't need the comment and it should instead say

`composer @sofisl @leahecole` 

inline with the others but in alphabetical order